### ISSUE TITLE
Unicode in directories

### ISSUE THREAD

**Issue by wisim** [beancount/beancount#547]
When I execute a commad like this:

`bean-web D:\Personal_Files\accounts\记账\test.bean`

I got an error:
```
INFO    : Reloading...
Traceback (most recent call last):
  File "d:\programdata\miniconda3\lib\site-packages\bottle.py", line 868, in _handle
    return route.call(**args)
  File "d:\programdata\miniconda3\lib\site-packages\bottle.py", line 1748, in wrapper
    rv = callback(*a, **ka)
  File "d:\programdata\miniconda3\lib\site-packages\beancount\web\web.py", line 1031, in wrapper
    entries, errors, options_map = loader.load_file(filename)
  File "d:\programdata\miniconda3\lib\site-packages\beancount\loader.py", line 89, in load_file
    entries, errors, options_map = _load_file(
  File "d:\programdata\miniconda3\lib\site-packages\beancount\loader.py", line 214, in wrapped
    result = function(toplevel_filename, *args, **kw)
  File "d:\programdata\miniconda3\lib\site-packages\beancount\loader.py", line 255, in _uncached_load_file
    return _load([(filename, True)], *args, **kw)
  File "d:\programdata\miniconda3\lib\site-packages\beancount\loader.py", line 503, in _load
    entries, parse_errors, options_map = _parse_recursive(
  File "d:\programdata\miniconda3\lib\site-packages\beancount\loader.py", line 394, in _parse_recursive
    src_options_map) = parser.parse_file(filename, encoding=encoding)
  File "d:\programdata\miniconda3\lib\site-packages\beancount\parser\parser.py", line 197, in parse_file
    _parser.parse_file(filename, builder, **kw)
OSError: Cannot open file 'D:\Personal_Files\accounts\记账\test.bean'
```

And my beancount version is 2.3.0 ( since 2.3.1 failed building on windows).

**Comment by dnicolodi** [comment]
This should be fixed in 2.3.1. Unfortunately I didn't catch the issue that prevent it to compile on Windows before the release. We should probably release 2.3.2 soon. Can you please try to build from the `v2` git branch and test again?

**Comment by wisim** [comment]
> 
> 
> This should be fixed in 2.3.1. Unfortunately I didn't catch the issue that prevent it to compile on Windows before the release. We should probably release 2.3.2 soon. Can you please try to build from the `v2` git branch and test again?

Thank you!

The current 2.3.2 version works great on my computer.  Nether of the mentioned issue exists. 


------
for anyone with the same issues, run:
1.  `git clone --branch v2 https://github.com/beancount/beancount.git`
2.  `cd beancount`
3.  `python setup.py install`

**Comment by dnicolodi** [comment]
Thank you for testing
